nservicebus/upgrades/7to8/index.md

```diff
@@ -134,3 +159,15 @@ The following transports might need migration:
 * Azure Storage Queues
 * SQL Transport
 * SQS
+
+## Outbox configuration
+
+NServiceBus version 8 requires explicit configuration of the transport transaction mode when using [Outbox](/nservicebus/outbox/):
+
+```csharp
+var transport = endpointConfiguration.UseTransport<MyTransport>();
+
+transport.TransportTransactionMode = TransportTransactionMode.ReceiveOnly;
```

Comment:
Does it make sense to specify, if I understood correctly, that `ReceiveOnly` is the only supported mode when using the Outbox?